docs/changes/units/16343.api.rst

```diff
@@ -0,0 +1,2 @@
+The ``factor`` parameter of the ``spectral_density`` equivalency is deprecated.
```

Comment:
should the changelog mention that it's been *effectively* deprecated for a while already ?

Reply:
Maybe something like "it was informally deprecated back in version 0.x (have to check that)" ?

Reply:
Agreed!